### PR TITLE
Honey Garlic Chicken

### PR DESCRIPTION
**Pull request by condog190**
Added honey garlic chicken with brown rice and broccoli dish.

### CODE REVIEW

**Review by DGKSK8LIFE — APPROVED** [review]

File: src/honey-garlic-chicken.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Honey Garlic Chicken with Broccoli and Brown Rice
+
+Very easy, high protein.
+
+- Servings: 2
+
+## Ingredients
+
+- 1/2 cup brown rice
```
Comment (siggines) [inline src/honey-garlic-chicken.md:9]:
```suggestion
- 1/2 cup (100g) brown rice
```

File: src/honey-garlic-chicken.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- 1/2 cup frozen peas
```
Comment (siggines) [inline src/honey-garlic-chicken.md:10]:
```suggestion
- 1/2 cup ( 70g ) frozen peas
```

**Review by siggines — CHANGES_REQUESTED** [review]

File: src/honey-garlic-chicken.md (hunk 2 above)
Comment (DGKSK8LIFE) [reply src/honey-garlic-chicken.md:10]: Keep the parenthesis formatting consistent. Should be `(70g)`.

Comment (siggines) [reply src/honey-garlic-chicken.md:10]: My mistake, above is now ( 100g ). Even tho you forgot to use the magic word @DGKSK8LIFE xD

**Pull request merged**